barretenberg/cpp/src/barretenberg/ultra_honk/relation_correctness.test.cpp

```diff
@@ -357,819 +356,4 @@ TEST_F(RelationCorrectnessTests, GoblinUltraRelationCorrectness)
     check_relation<Flavor, std::tuple_element_t<6, Relations>>(circuit_size, prover_polynomials, params);
     check_linearly_dependent_relation<Flavor, std::tuple_element_t<7, Relations>>(
         circuit_size, prover_polynomials, params);
-}
-
-/**
- * @brief Test the correctness of GolbinTranslator's Permutation Relation
- *
- */
-TEST_F(RelationCorrectnessTests, GoblinTranslatorPermutationRelationCorrectness)
```

Comment:
where have these tests disappeared?

Reply:
Discussed ofline, they moved to another file having the same name but in the translator folder.